pyanaconda/modules/payloads/source/live_os/initialization.py

```diff
@@ -96,6 +100,22 @@ def __init__(self, image_path, target_mount):
         super().__init__(target_mount)
         self._image_path = image_path
 
+    def run(self):
+        """Run the task.
+
+        Calculates size of the live image image.
+        """
+        super().run()
+
+        required_space = self._calculate_required_space()
+        log.debug("Size of live OS image payload: %s.", Size(required_space))
```

Comment:
This can say just `Required space: `.  In the logs, it should be clear that this is a Live OS image from the context. Please, remove the dot after the size and move the logging to the `_calculate_required_space` method.